### PR TITLE
ICAPS Hierarchical Planning initialization

### PR DESCRIPTION
**Pull request by pmandler**

### CODE REVIEW

File: venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/config.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    builder.set_homepage_header({
+    'title': 'Second Workshop on Hierarchical Planning',
+    'subtitle': 'ICAPS 2019 Workshop',
+    'deadline': 'Submission Deadline: March 24, 2019 midnight AoE',
+    'date': 'July 11-15, 2019',
+    'website': 'https://icaps19.icaps-conference.org/workshops/Hierarchical-Planning/index.html',
+    'location': 'Berkeley, CA, USA'
+    })
+    builder.set_double_blind(True)
+    builder.set_submission_public(True)
```
Comment (melisabok) [inline venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/config.py]: I think they asked to remain everything private

File: venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/init.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+conference = config.get_conference(client)
+# March 24 midnight anywhere on earth
+conference.open_submissions(due_date = datetime.datetime(2019, 3, 25, 11, 59))
```
Comment (melisabok) [inline venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/init.py]: I think it is 10:59.

File: venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/config.py (hunk 1 above)
Comment (pmandler) [reply venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/config.py]: You're right. They wanted only accepted papers published.

File: venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/init.py (hunk 2 above)
Comment (pmandler) [reply venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/init.py]: I disagree.  On the web it says it's 12 hours off.  Show me where it's 11 hours difference and I'll change it.  Actually I should change it to 12:00.

Comment (melisabok) [reply venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/init.py]: You are right. I was confused to Samoa time, that is -11/

**Pull request merged**